web-components/src/components/button/button.stories.ts

```diff
@@ -54,73 +54,95 @@ const Template: StoryFn<Button> = ({
   </dss-button>
 `;
 
-export const Default = Template.bind({});
-Default.args = {
-  slot: 'Primary Button',
+export const Default: StoryObj<Button> = {
+  render: Template,
+  args: {
+    slot: 'Primary Button',
+  },
```

Comment:
**praise:** these are so much nicer and cleaner

Is this due to a Storybook update or could have had it this way all along?

Reply:
This new format was released at some point towards the end of last year

Reply:
I also think it's nice and clean. @Cybertron01Z mentioned it first :)